lotto645.py

```
@@ -163,100 +192,143 @@ def _try_buying(self, headers: dict, data: dict) -> dict:
+            res = self.http_client.get(api_url, params=params, headers=headers)
+            
+            if res.status_code != 200:
+                print(f"DEBUG: API Status {res.status_code}")
+                pass
+            
+            try:
+                data = res.json()
+                data = data.get("data", {})
+                if "list" not in data:
+                    print("DEBUG_DATA_LIST_MISSING_IN_DATA")
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)